bizhawk-crowd-shuffler.lua

```diff
@@ -50,7 +51,7 @@ local function parseAndExecuteResponse(response)
             args = t[2]
         }
 
-        -- print("DEBUG: command=" .. input.command)
+        print("DEBUG: command=" .. input.command)
```

Comment:
Might want to double check and make sure to comment this again.

Reply:
Ah, I'm currently hunting down occasional crashes on loading a save state. Though it might just be wierd savestate corruption.